### PR TITLE
Add config for eslint

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,388 @@
+{
+  "env": {
+    "es6": true,
+    "node": false
```
Comment (vergilius) [inline .eslintrc]: aren't we using the same linter for node apps? I mean, in the future ;P

Comment (aju) [inline .eslintrc]: This is the question - OK I will enable it for now

**Comment by vergilius** [comment]
I wonder, because many of the rules are duplicated, maybe would be good idea to 'extend' it somehow to prevent duplication? I remember that there was build in solution

**Comment by aju** [comment]
It would be harder to set it up with project as long as we keep in in sync It should be fine

**Comment by vergilius** [comment]
mhm, :+1:

**Comment by matzimowski** [comment]
:+1:

**Comment by kdzwinel** [comment]
:+1:

**Pull request merged**